beacon-chain/blockchain/service.go

```diff
@@ -195,7 +195,7 @@ func (s *Service) initializeBeaconChain(
 	eth1data *ethpb.Eth1Data) error {
 	_, span := trace.StartSpan(context.Background(), "beacon-chain.Service.initializeBeaconChain")
 	defer span.End()
-	log.Info("Genesis time reached, starting the beacon chain")
```

Comment:
Fixed this log where it says genesis time reached but it hasn't reach. The log is meant for initializing beacon genesis state